TACC/Core-CMS-Custom
# feat(example): sample cms & example app

## Overview

Add an `example-cms` with an example custom app.
<sup>So a developer of a new custom CMS need not initially tweeze code from existing custom CMS's to get started.</sup>

## Related

N/A

## Changes

- add `/example-cms`
- delete `/apcd-cms/src/apps/custom_example`

## Testing

1. Follow `/README.md` to set up and run CMS.
2. Open https://localhost:8000.
3. Verify CMS loads.
4. Open https://localhost:8000/example.
5. Verify CMS test app loads.

## UI

…

## Notes

Use of this directory is documented at [Django CMS - Developer Guide - Customize a New CMS](https://confluence.tacc.utexas.edu/x/5JB9E).
